daemon/cmd/daemon_main.go

```diff
@@ -1981,6 +1981,19 @@ func startDaemon(d *Daemon, restoredEndpoints *endpointRestoreState, cleaner *da
 	bootstrapStats.updateMetrics()
 	go d.launchHubble()
 
+	// Start controller to validate daemon config is unchanged
```

Comment:
Not sure this is going to work, see earlier comment

Reply:
Will fix. I did not inspect the controller status for errors yet, though. Do you happen to know of a code in the CI that could be used to surface controller errors?

Reply:
Ginkgo suite checks for controller errors in Agent preflight checks on `ExpectCiliumReady()`, so there is some coverage, but none in the `cilium-cli` test suite.